lib/functoria/impl.ml

```diff
@@ -185,8 +185,7 @@ let rec equal : type t1 t2. t1 t -> t2 t -> (t1, t2) Typeid.witness =
 
 and equal_abstract (Abstract x) (Abstract y) = Typeid.to_bool @@ equal x y
 
-and equal_tl :
-    type t1 t2 v1 v2.
+and equal_tl : type t1 t2 v1 v2.
```

Comment:
Why is this needed?

Reply:
It's only a cosmetic change. These can make the type annotation harder to read when it all fits on one line and the `and equal_tl :` line is always short when the type is long enough.